src/index_enable_billing_activation.js

```diff
@@ -1,55 +1,106 @@
+    type: params.type,
     awsDesc: params.desc,
     activationDate: datetime
-  }
+  };
+  const billingUrl = `${params.billingUrl}?client_id=${params.apiKey}&client_secret=${params.secretKey}`;
+  console.log(billingUrl);
   console.log(bodyjson);
-  var billingUrl = params.billingUrl+"?client_id="+ params.apiKey +"&client_secret="+params.secretKey;
-  req({
-    url: billingUrl, //URL to hit
-    method: 'POST',
-    json: bodyjson 
-    }, function(error, response, body){
-           console.log("error:");
-           console.log(error);
-           console.log("response:");
-           console.log(response.body);
-  });
+  try {
+     req({url: billingUrl, method: 'POST',json: bodyjson },(error, response, body) => {
+        console.log(`error:  ${JSON.stringify(error)}`);
+        console.log(`response: ${JSON.stringify(response.body)}`);
+        console.log(`body: ${JSON.stringify(body)}`);
+        console.log(`body: ${JSON.stringify(response.statusCode)}`);
+        if (error) {
+          sendAlertMailBySes(bodyjson, error, params);
+        } else if (response.body.errorcode == '0') {
+          console.log('successfully registered with billing system');
+        } else {
+          sendAlertMailBySes(bodyjson, response.body, params);
+        }
+     });
+  } catch (err) {
+    sendAlertMailBySes(bodyjson, err, params);
+  }
 }
-
+//----------------------------------------------------------------------
 exports.handler = (event, context, callback) => {
-
   // find account id
-  var accountId = null;
-  var retDoc = event.account.result.body;
-  var billing = event.account.billingDetails;
-  if (typeof(retDoc) == 'string') {
+  console.log(event);
+  let accountId = null;
+  let retDoc = event.account.result.body;
+  const billingInfo = event.account.billingDetails;
+  if (typeof retDoc === 'string') {
     retDoc = JSON.parse(retDoc);
   }
-  if (retDoc.Account) {
-    // this is result of find account
-    accountId = retDoc.Account.Id;
-  }
-  else {
-    // this is result of create account
-    accountId = retDoc.CreateAccountStatus.AccountId;
-  }
-  console.log(retDoc);
-  if(accountId){
-    registerAccount(accountId,billing);
-    event.IntegrateBillingStatus=true;
-  }else{
-    console.log("Customer Accound not found");
-    event.IntegrateBillingStatus=false;
+  if (retDoc) {
+    if (retDoc.Account) {
+      // this is result of find account
+      accountId = retDoc.Account.Id;
+    } else {
+      // this is result of create account
+      accountId = retDoc.CreateAccountStatus.AccountId;
+    }
+    if (accountId) {
+      // sending request to billing system(rest call) for register account
+      registerAccount(accountId, billingInfo);
+    } else {
+      console.log(`customer account id not found: ${JSON.stringify(retDoc)}`);
+      const errmsg = `Found Error While Creating Account:  ${JSON.stringify(retDoc.CreateAccountStatus.FailureReason)}`;
+      sendAlertMailBySes(billingInfo, errmsg, billingInfo);
```

Comment:
no any requirement . I did additional just for tracking but any way i can comment code for send alert mail